content/en/community/hackathons/sessions/testing-unikraft/sol/prepare.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Check the scripts with [shellcheck](https://www.shellcheck.net/), no major issue, just some double quotes to prevent word splitting.